chronix-server-query-language/src/main/java/de/qaware/chronix/cql/antlr/CQLCFParser.java

```diff
@@ -62,7 +64,7 @@
     public static final ATN _ATN =
             new ATNDeserializer().deserialize(_serializedATN.toCharArray());
     protected static final DFA[] _decisionToDFA;
-    protected static final PredictionContextCache _sharedContextCache =
+    private final PredictionContextCache sharedContextCache =
```

Comment:
\[Checkstyle\] ERROR: Variable '\_sharedContextCache' must be private and have accessor methods\.